packages/entities/src/lib/update.mutation.spec.ts

```diff
@@ -203,4 +190,35 @@ describe('update', () => {
       expect(ids).toBe(sameIds);
     });
   });
+
+  describe('UpdateEntitiesIds', () => {
+    it('should support id update', () => {
+      store.update(addEntities([createTodo(1)]));
+      toMatchSnapshot(expect, store, 'id updated false');
+      store.update(updateEntitiesIds(1, 2));
+      toMatchSnapshot(expect, store, 'id updated true');
+    });
+
+    it('should update multiple ids', () => {
+      store.update(addEntities([createTodo(1), createTodo(2), createTodo(3)]));
+      toMatchSnapshot(expect, store, 'ids updated false');
+      store.update(updateEntitiesIds([2, 3], [4, 5]));
+      toMatchSnapshot(expect, store, 'ids updated true');
+    });
+
+    it('should throw if new id already exists in the store', () => {
+      store.update(addEntities([createTodo(1), createTodo(2)]));
+      expect(() => {
+        store.update(updateEntitiesIds(1, 2));
+      }).toThrow();
+    });
+
+    it('should work with ref', () => {
+      const store = createUIEntityStore();
+      store.update(addEntities([createUITodo(1)], { ref: UIEntitiesRef }));
+      toMatchSnapshot(expect, store, 'id updated false');
+      store.update(updateEntitiesIds(1, 2, { ref: UIEntitiesRef }));
```

Comment:
add a test that updated the entity after changing the id, please

Reply:
will do